src/components/InfoPopup/InfoPopup.js

```diff
@@ -0,0 +1,49 @@
+// Context help component: an icon of an i within a circle that raises a popup on
+// mouseover. Intended to be placed next to components and controls to explain their 
+// function and usage.
+// Each instance has a unique "index", which matches an attribute in public/help.yaml.
+// The yaml file provides the title and text to be displayed in the popup. 
+
+import React from "react";
+import PropTypes from "prop-types";
+
+import "./InfoPopup.css";
+import { OverlayTrigger, Popover } from "react-bootstrap";
+import { InfoCircle } from "react-bootstrap-icons";
+import useStore from '../../store/useStore.js'
+
+
+
+function InfoPopup({
+  label = <InfoCircle />,
+  placement = "right",
+  index
+}) {
+
+  const helpText = useStore((state) => state.helpText);
+
+  return (
+    <span className="InfoPopup">
+      <OverlayTrigger
+        key={`${index}-help-popup`}
+        placement={placement}
+        overlay={
+          <Popover id={helpText[index] ? helpText[index].title : "Loading"}>
```

Comment:
Expressions like this can get shorter and easier to read by using the [optional chaining operator `?.`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) and the [nullish coalescing operator `??`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing) (terrible name; means well-behaved version of `||`). 

This one would become `helpText[index]?.title ?? "Loading"`. 

A really convenient thing about `?.` is that it can be used repeatedly in the same expression: `a?.b.?.c?.d`, and it returns undefined if any prop doesn't exist. So much better than the equivalent ternary `?` expression.